serverless/cluster-logging-serverless.adoc

```diff
@@ -0,0 +1,11 @@
+[id="cluster-logging-serverless"]
```

Comment:
please update the topic yml file and add this assembly as per Brandi's comment - there won't be a preview available until this is added and not commented out